jstests/core/explain_verbosity_unknown.js

```diff
@@ -0,0 +1,22 @@
+//Test whether unknown commands are accepted in explain function
+
+(function() {
+	
+	"use strict";
+	
+	var unknownCollection = db["jstestDB"];
```

Comment:
We typically use a collection name that is identical to that of the test. In this case, I'd suggest calling the test 'jstests/core/explain_cmd_unknown_options.js', since it's a little more descriptive. You can also use '.' syntax here, and should prefer using 'let' over 'var', since 'let' has slightly better scoping rules: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let:

> let allows you to declare variables that are limited in scope to the block, statement, or expression on which it is used. This is unlike the var keyword, which defines a variable globally, or locally to an entire function regardless of block scope.

Thus, this line should read `let collection = db.explain_cmd_unknown_options;`